config/locales/client.en.yml

```
@@ -1930,6 +1930,10 @@ en:
         title: "Keep reading!"
         content: "Here are some topics we think you might like to read next."
 
+      welcome_topic:
+        title: "Welcome topic"
+        content: "As an admin, you can edit this welcome topic and make it more welcoming to your community members!"
```

There's a bit of repetition here (also with the title of topic). Maybe a small copyedit can help: 

```suggestion
        title: "Edit the welcome topic"
        content: "Help new members feel at home by customizing this topic to suit your community's needs."
```